.env.example

```diff
@@ -0,0 +1,4 @@
+export SOCIAL_AUTH_GOOGLE_OAUTH2_KEY=EXAMPLE
```

Comment:
Remove this file

Reply:
Could you elaborate?

Reply:
This file should be deleted. We also dont use environment variables for passing prod settings